packs/spells/corrosive-body.json

```diff
@@ -18,17 +18,47 @@
             "value": ""
         },
         "damage": {
-            "value": {}
+            "value": {
+                "Pb1eT1upkmw6q6JG": {
+                    "applyMod": false,
+                    "type": {
+                        "categories": [],
+                        "subtype": "",
+                        "value": "acid"
+                    },
+                    "value": "3d6"
+                }
+            }
```

Comment:
I think I would rather just remove the damage roll from fiery body than add it here.